Release: Gateway 3.9.1.0 (#8536)
Kong/docs.konghq.com: app/gateway/changelog.md

For product versions that have reached the end of sunset support, see the [changelog archives](https://legacy-gateway--kongdocs.netlify.app/enterprise/changelog/).

## 3.9.1.0
**Release Date** 2025/03/11

### Fixes
#### Core

> * Fixed an issue where a mismatch between If-Match in the request and ETag in the response would result in a bad case in the response phase.0)

Reviewer: ```suggestion
* Fixed an issue where a mismatch between If-Match in the request and ETag in the response would result in a bad case in the response phase.
```
Wasn't sure if that was correct or not

* Updated the AWS Vault supported regions list to the latest available.
* Added support for the new Ollama streaming content type in the AI driver.

#### Plugins

* [**AI Proxy**](/hub/kong-inc/ai-proxy/) (`ai-proxy`)
  * Fixed an issue with Gemini streaming responses getting truncated and/or missing tokens.
  * Fixed an incorrect error thrown when trying to log streaming responses.
  * Fixed an issue where tool calls weren't working in streaming mode for Bedrock and Gemini providers.

* [**AI Semantic Prompt Guard**](/hub/kong-inc/ai-semantic-prompt-guard/) (`ai-semantic-prompt-guard`)

>   *  Fixed an issue where Kong Gateway was not able to reconfigure the plugin when using DB-less mode.

Reviewer: ```suggestion
  *  Fixed an issue where Kong Gateway couldn't reconfigure the plugin when using DB-less mode.
```
